edx_sga/sga.py

```diff
@@ -13,26 +13,36 @@
 
 from functools import partial  # lint-amnesty, pylint: disable=wrong-import-order
 
-from courseware.models import StudentModule  # lint-amnesty, pylint: disable=import-error
+try:
```

Comment:
We shouldn't silence `ImportError`s here. Instead can you `patch` these modules in the tests?

Reply:
@noisecapella any suggestion how?

Reply:
Add this to `StaffGradedAssignmentMockedTests.setUp`:


        try:
            import builtins
        except ImportError:
            import __builtin__ as builtins
        real_import = builtins.__import__

        def fake_import(*args, **kwargs):
            try:
                return real_import(*args, **kwargs)
            except ImportError:
                return mock.Mock()

        builtins.__import__ = fake_import

        def restore_import():
            builtins.__import__ = real_import

        self.addCleanup(restore_import)

This will replace the import function with one which silences import errors. This isn't ideal but this way it's confined to the test code instead of showing up in the implementation